### PR TITLE
chore(ci): send event on new tag

### PR DESCRIPTION
**Pull request by MarcoIeni**
Send an event when a new tag is released.
Accelerate needs this event to build the docker image of the Query Engine.

- [x] set secret

### CODE REVIEW

File: .github/workflows/send-tag-event.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - run: echo "Sending event for tag ${{ github.ref_name }}"
+      - uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.ACCELERATE_DOCKER_QE_BUILD }}
+          script: |
+            await github.rest.repos.createDispatchEvent({
+              owner: 'prisma',
+              repo: 'pdp-cloudflare',
+              event_type: "orm_release",
```
Comment (MarcoIeni) [inline .github/workflows/send-tag-event.yml:20]:
```suggestion
              event_type: "engines_release",
```

File: .github/workflows/send-tag-event.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: Send Tag Event
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-and-push:
```
Comment (janpio) [inline .github/workflows/send-tag-event.yml:9]: I think nothing is being built and pushed here.

Comment (MarcoIeni) [inline .github/workflows/send-tag-event.yml:9]:
```suggestion
  send-tag-event:
```

Comment (MarcoIeni) [reply .github/workflows/send-tag-event.yml]: oops, copy paste issue, fixed 👍

File: .github/workflows/send-tag-event.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: Send Tag Event
+
+on:
+  push:
+    tags:
+      - '*'
```
Comment (Jolg42) [inline .github/workflows/send-tag-event.yml:6]:
```suggestion
      - '!*-integration'
```

If you only want to receive dev and latest tags.

**Review by Jolg42 — APPROVED** [review]
Nice!

**Review by millsp — APPROVED** [review]

File: .github/workflows/send-tag-event.yml (hunk 3 above)
Comment (MarcoIeni) [reply .github/workflows/send-tag-event.yml:6]: if we don't see values in using the integration tags with accelerate, then yeah, let's remove them!

**Comment by MarcoIeni** [comment]
Thanks for the review!
Jan just set the secret 👍 
Can anyone merge this? 🙏

**Pull request merged**